implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -128,34 +128,30 @@ pub(crate) fn multiaddr_to_route(ma: &MultiAddr) -> Option<Route> {
 }
 
 pub fn setup_logging(verbose: u8) {
-    let ockam_crates = [
-        "ockam",
-        "ockam_node",
-        "ockam_core",
-        "ockam_command",
-        "ockam_identity",
-        "ockam_channel",
-        "ockam_transport_tcp",
-        "ockam_vault",
-        "ockam_vault_sync_core",
-    ];
     let builder = EnvFilter::builder();
-    let filter = match env::var("OCKAM_LOG") {
-        Ok(s) if !s.is_empty() => builder.with_env_var("OCKAM_LOG").from_env_lossy(),
-        _ => match verbose {
-            0 => builder
-                .with_default_directive(LevelFilter::WARN.into())
-                .parse_lossy(ockam_crates.map(|c| format!("{c}=info")).join(",")),
-            1 => builder
-                .with_default_directive(LevelFilter::INFO.into())
-                .parse_lossy(""),
-            2 => builder
-                .with_default_directive(LevelFilter::DEBUG.into())
-                .parse_lossy(""),
-            _ => builder
-                .with_default_directive(LevelFilter::TRACE.into())
-                .parse_lossy(""),
+    // If `verbose` is not set, try to read the log level from the OCKAM_LOG env variable.
+    // If both `verbose` and OCKAM_LOG are not set, logging will not be enabled.
+    // Otherwise, use `verbose` to define the log level.
+    let filter = match verbose {
+        0 => match env::var("OCKAM_LOG") {
+            Ok(s) if !s.is_empty() => builder.with_env_var("OCKAM_LOG").from_env_lossy(),
+            _ => return,
         },
+        1 => builder
+            .with_default_directive(LevelFilter::ERROR.into())
+            .parse_lossy(""),
+        2 => builder
+            .with_default_directive(LevelFilter::WARN.into())
+            .parse_lossy(""),
+        3 => builder
+            .with_default_directive(LevelFilter::INFO.into())
+            .parse_lossy(""),
+        4 => builder
+            .with_default_directive(LevelFilter::DEBUG.into())
+            .parse_lossy(""),
+        _ => builder
+            .with_default_directive(LevelFilter::TRACE.into())
+            .parse_lossy(""),
```

Comment:
I think a single `-v` should be enough to enable `INFO` level logging. WARN and ERROR events by themselves are almost useless to see what is going on and INFO is low-frequency enough that it can still be considered "mostly quiet"